entrypoint.sh

```diff
@@ -35,20 +35,6 @@ time_format=''
 log() {
     :
 }
-if [[ -v DEBUG ]]; then
```

Comment:
Any particular reason to remove this? Not a blocker, we can always bring it back later.

Reply:
Nope, it didn't work with my shell on the Mac so had removed it early when I was testing. I added it back.